plugins/lsp/language-servers/rust-analyzer.nix

```diff
@@ -57,6 +67,7 @@ in
     extraPackages =
       with pkgs;
       (optional ((isBool cfg.installCargo) && cfg.installCargo) cfg.cargoPackage)
-      ++ (optional ((isBool cfg.installRustc) && cfg.installRustc) cfg.rustcPackage);
+      ++ (optional ((isBool cfg.installRustc) && cfg.installRustc) cfg.rustcPackage)
+      ++ (optional ((isBool cfg.installRustfmt) && cfg.installRustfmt) cfg.rustfmtPackage);
```

Comment:
It pains me to see so many unnecessary parentheses, but it matches the style of the existing code 😅

Just to illustrate a point, this is equivalent to:
```nix
optional (isBool cfg.installCargo && cfg.installCargo) cfg.cargoPackage
++ optional (isBool cfg.installRustc && cfg.installRustc) cfg.rustcPackage
++ optional (isBool cfg.installRustfmt && cfg.installRustfmt) cfg.rustfmtPackage;
```